using-mobile-extensions/adobe-target/target-event-reference.md

```diff
@@ -120,6 +120,7 @@ The Target extension will read the following keys from the configuration event:
 | `target.environmentId` | String | Yes | The Target Environment ID. |
 | `target.timeout` | Integer | Yes | This value is used for Target request connection timeouts \(in seconds\). If the value is not set, a default value of 5 seconds is used. |
 | `global.privacy` | String | Yes | This contains the mobile privacy status settings. |
+| `target.previewEnabled` | Boolean | Yes | Boolean value to determine if Target Preview should be enabled or not. If not set, then Preview is enabled by default. |
```

Comment:
- Please replace "if" with "whether".
- Remove "or not"

Reply:
done